lib/CMakeLists.txt

```
@@ -47,20 +47,24 @@ if(USE_ARES)
   include_directories(${CARES_INCLUDE_DIR})
 endif()
 
-add_library(
-  curlu # special libcurlu library just for unittests
-  STATIC
-  EXCLUDE_FROM_ALL
-  ${HHEADERS} ${CSOURCES}
-)
-target_compile_definitions(curlu PUBLIC UNITTESTS CURL_STATICLIB)
+if (BUILD_TESTING)
+    add_library(
+      curlu # special libcurlu library just for unittests
+      STATIC
+      EXCLUDE_FROM_ALL
+      ${HHEADERS} ${CSOURCES}
+    )
+    target_compile_definitions(curlu PUBLIC UNITTESTS CURL_STATICLIB)
+endif()
```

```suggestion
if(BUILD_TESTING)
  add_library(
    curlu # special libcurlu library just for unittests
    STATIC
    EXCLUDE_FROM_ALL
    ${HHEADERS} ${CSOURCES}
  )
  target_compile_definitions(curlu PUBLIC UNITTESTS CURL_STATICLIB)
endif()
```
Let's keep indenting by 2 spaces.